components/inputs/input-date.js

```diff
@@ -273,8 +273,7 @@ class InputDate extends LocalizeStaticMixin(LitElement) {
 			await this._handleChange();
 			this._calendar.focus();
 			this._setFormattedValue();
-
-			if (e.keyCode === 40) e.preventDefault();
+			e.preventDefault();
```

Comment:
there was a bug introduced where if enter was pressed to open the calendar on the input, the first date (e.g., today) would just be selected (dropdown would never be seen open, the text box would just fill)